x-pack/docs/en/rest-api/security/get-api-keys.asciidoc

```diff
@@ -15,10 +15,16 @@ Retrieves information for one or more API keys.
 [[security-api-get-api-key-prereqs]]
 ==== {api-prereq-title}
 
-* To use this API, you must have at least the `manage_own_api_key` or the `read_security`
-cluster privileges.
-* If you have only the `manage_own_api_key` privilege, this API returns only
-the API keys that you own. If you have `read_security`, `manage_api_key` or greater
+* To use this API, you must have at least the `manage_own_api_key` or the 
+`read_security` cluster privileges.
+** If you only have the `manage_own_api_key` privilege, this API only returns
+the API keys that you own. 
++
+NOTE: Authenticating with an API key that has the `manage_own_api_key` privilege
+does not allow retrieving the authenticated user's own keys using `owner=true`.
+Instead, specify a key ID in the `id` field, or authenticate the user with basic
```

Comment:
Nit: I don't think the "specify a key ID in the `id` field" part quite fits here since it still doesn't allow you to fetch the user's own API keys. The long, clunky version of this the relevant bits is something like:

"Authenticating with an API key that has the `manage_own_api_key` privilege
does not allow retrieving the authenticated user's own keys. Instead, authenticate the user with basic credentials. Any API key can still be used to retrieve information about itself, by specifying its ID in the `id` field." 

I think it may be clearest to just leave out the "Any API key can still be used to retrieve information about itself, by specifying its ID in the `id` field." bit entirely since it only indirectly pertains here. WDYT?

Reply:
Ah, yes. I was confused and thought a user would own the key with which they authenticate, but I see now that's not the case. 

I've pushed a change with your suggestions (drop the _"using `owner=true`"_) and the sentence "_Any API key can still be used to retrieve information about itself, by specifying its ID in the id field."_